### DISCUSSION TITLE
How to make relation annotation ?

### DISCUSSION THREAD

**Discussion by gegerihu** [doccano/doccano#1877] [Q&A]
I pulled the latest version of doccano, and created a sequence annotation project, but it can't annotate the relationship between entities. Could someone know how to fix it ?

**Comment by gegerihu** [comment]
When creating a project which has to click relation labeling option.

**Comment by david-engelmann** [comment]
You should see in the label section of doccano two categories, span and relation. Here you can add the labels for the spans and the labels for relations.

![image](https://user-images.githubusercontent.com/81196676/199319379-c3359f49-a186-4c65-929c-f85f5afe7e48.png)

When you are in the annotating screen, you should see the span labels you've created and a toggle to see the relation labels you've created.

![image](https://user-images.githubusercontent.com/81196676/199319707-32f3f363-4793-4f5e-a81e-afe35b62e6b9.png)

once you've labeled the spans you can connect them with relation labels by clicking the toggle so it's in relation mode, select the relation label you would like to add. To add the relation you can click on the span labels, if the direction of the arrow is intentional, make sure you click on the parent span first, the second span you click on will have the arrow pointing to it.

In this example, I click on "where" first then "I get"
![image](https://user-images.githubusercontent.com/81196676/199321903-a7b5ed83-17ac-4cb5-be51-e19eccdc3171.png)

![image](https://user-images.githubusercontent.com/81196676/199322211-7a20d55b-ad1a-4d06-83f7-9f11f554c099.png)

I get this relation displayed
![image](https://user-images.githubusercontent.com/81196676/199322373-8b6bc195-8b01-4932-9de1-02df3b0ccf27.png)

The labels in this example are just for mechanics purposes and don't represent a good label or relation
